### PR TITLE
Remove wishlist

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it removes wishlist from the storefront, as requirements has been changed and it has not yet finished API and functionality.

### CODE REVIEW

File: src/@sdk/index.ts [hunk 1]
```diff
@@ -62,122 +67,104 @@ export const createSaleorClient = (url?: string, cache = new InMemoryCache()) =>
   });
 
 export class SaleorAPI {
-  getAttributes = this.watchQuery(QUERIES.Attributes, data => data.attributes);
+  getAttributes = this.watchQuery(
```
Comment (krzysztofwolski) [inline src/@sdk/index.ts:70]: Can we just remove whats not longer needded without introducing lots of code style changes?

Comment (orzechdev) [reply src/@sdk/index.ts:70]: No idea why this change occured... I'll investigate it

Comment (krzysztofwolski) [reply src/@sdk/index.ts:70]: Updated vscode plugin and format on save probably

Comment (orzechdev) [reply src/@sdk/index.ts]: yes, exactly 👍

**Pull request merged**